Makefile.in

```diff
@@ -50,3 +53,9 @@ install: all
 
 distcheck:
 	@$(ECHO) @PACKAGE_VERSION@
+
+dist:
+	autoconf
+	tar -s +^+msktutil-@PACKAGE_VERSION@/+ -zcvf msktutil-@PACKAGE_VERSION@.tgz \
+           $(SOURCES) $(HEADERS) configure README LICENSE ChangeLog INSTALL \
+           Makefile.in configure.in config.h.in msktutil.M
```

Comment:
I'd suggest to add these extra files (except maybe the auto-generated `configure` and `config.h.in`) the a new variable `EXTRA_DIST`. (Or just sod it, and use `automake` as well.)